### ISSUE TITLE
Incorrect error message for "stroustrup" brace-style

### ISSUE THREAD

**Issue by burnnat** [eslint/eslint#719]
When using the "stroustrup" for the brace-style rule, code like the following:

```
if (foo) {
  bar();
} else {
  baz();
}
```

Generates an error that says:

> Closing curly brace does not appear on the same line as the subsequent block.

But in fact, the closing curly brace _does_ appear on the same line as the subsequent block. To accurately convey the situation, the error message should be the opposite of the "1tbs" style message:

> Closing curly brace appears on the same line as the subsequent block.
